guacamole/src/main/webapp/app/login/styles/dialog.css

```diff
@@ -115,13 +115,6 @@
     background-image: url("images/guac-tricolor.png");
 }
 
-.login-ui.continuation div.login-dialog {
-    border-right: none;
-    border-left: none;
-    box-shadow: none;
-    max-width: 6in;
-}
-
```

Comment:
Ah, OK. Well, this doesn't look bad to me (though that text is a bit squished). Based on the screenshot, I don't have any major objection to the style changes if they are necessary.